ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -813,6 +813,70 @@
         ]
       }
     },
+    {
+      "name": "stream-stream-stream - outer-outer",
+      "statements": [
+        "CREATE STREAM S1 (ID INT KEY, V0 bigint) WITH (kafka_topic='left', value_format='JSON');",
+        "CREATE STREAM S2 (ID INT KEY, V0 bigint) WITH (kafka_topic='right', value_format='JSON');",
+        "CREATE STREAM S3 (ID INT KEY, V0 bigint) WITH (kafka_topic='right2', value_format='JSON');",
```

Comment:
`right` and `right2` is confusing... Can we rename the topics to `left_topic`, `middle_topic` and `right_topic` (make reading the `"inputs"` section below easier.

Reply:
This follows the pattern of all the other tests in this file. For this comment I actually think consistency is more important for readability than the rename so I've left it as is. If we want to go back through and update all the tests in this file we can do that in one big batch down the road.